Source/WebKit/UIProcess/ios/WKTextInteractionWrapper.mm

```diff
@@ -56,7 +56,9 @@ - (instancetype)initWithView:(WKContentView *)view
         [view addInteraction:_asyncTextInteraction.get()];
     } else
 #endif // HAVE(UI_ASYNC_TEXT_INTERACTION)
+    {
         _textInteractionAssistant = adoptNS([[UIWKTextInteractionAssistant alloc] initWithView:view]);
+    }
```

Comment:
I prefer this over adding the `IGNORE_WARNINGS` macros because the `if/else` code turned off on some platforms.

But it's technically a violation of the WebKit coding standard, so I'll change both to match—whatever the reviewer(s) decide.